Index: mplayer-dmo-1.0.rc2svn20090508/libao2/ao_jack.c
===================================================================
--- mplayer-dmo-1.0.rc2svn20090508.orig/libao2/ao_jack.c	2009-05-08 12:05:16.000000000 +0200
+++ mplayer-dmo-1.0.rc2svn20090508/libao2/ao_jack.c	2009-05-08 12:05:26.000000000 +0200
@@ -51,7 +51,7 @@
 LIBAO_EXTERN(jack)
 
 //! maximum number of channels supported, avoids lots of mallocs
-#define MAX_CHANS 6
+#define MAX_CHANS 128
 static jack_port_t *ports[MAX_CHANS];
 static int num_ports; ///< Number of used ports == number of channels
 static jack_client_t *client;
